prometheus-exporter/src/main/java/be/xplore/githubmetrics/prometheusexporter/PrometheusExporter.java

```diff
@@ -0,0 +1,33 @@
+package be.xplore.githubmetrics.prometheusexporter;
+
+import be.xplore.githubmetrics.domain.domain.WorkflowRun;
+import be.xplore.githubmetrics.domain.usecases.ports.out.WorkflowRunsExportPort;
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import org.springframework.stereotype.Service;
+
+import java.util.EnumMap;
+import java.util.Map;
+
+@Service
+public class PrometheusExporter implements WorkflowRunsExportPort {
+
+    private final MeterRegistry registry;
+
+    public PrometheusExporter(MeterRegistry registry) {
+        this.registry = registry;
+    }
+
+    @Override
+    public void exportWorkflowRunsStatusCounts(EnumMap<WorkflowRun.RunStatus, Integer> statuses) {
+        for (Map.Entry<WorkflowRun.RunStatus, Integer> entry : statuses.entrySet()) {
+            Gauge.builder(
+
```

Comment:
Nitpicky, but this blank line seems to not be needed

Reply:
I'm fine with nitpicky. also fixed